plugins/plugin-svn/che-plugin-svn-ext-ide/src/main/java/org/eclipse/che/plugin/svn/ide/SubversionClientService.java

```diff
@@ -161,12 +174,23 @@
      *         whether or not to ignore externals (--ignore-externals)
      * @param accept
      *         the accept argument (--accept)
+     * @param userName
+     *         the user name for authentication
+     * @param password
+     *         the password for authentication
      */
-    Promise<CLIOutputWithRevisionResponse> update(Path project, Path[] paths, String revision, String depth, boolean ignoreExternals, String accept);
+    Promise<CLIOutputWithRevisionResponse> update(Path project,
+                                                  Path[] paths,
+                                                  String revision,
+                                                  String depth,
+                                                  boolean ignoreExternals,
+                                                  String accept,
+                                                  @Nullable String userName,
+                                                  @Nullable String password);
 
     Promise<CLIOutputResponse> showLog(Path project, Path[] paths, String revision);
 
-    Promise<CLIOutputResponse> showDiff(Path project, Path[] paths, String revision);
+    Promise<CLIOutputResponse> showDiff(Path project, Path[] paths, String revision, @Nullable String userName, @Nullable String password);
```

Comment:
Here you use @Nullable for username and password but you don't use it for info method. Why? Let's do in the same way